ipfx/bin/run_feature_extraction.py

```diff
@@ -47,7 +47,12 @@ def run_feature_extraction(input_nwb_file,
     if len(sweep_info) == 0:
         raise er.FeatureError("There are no QC-passed sweeps available to analyze")
 
-    ont = StimulusOntology(ju.read(stimulus_ontology_file)) if stimulus_ontology_file else StimulusOntology()
+    if stimulus_ontology_file:
+        mso.make_stimulus_ontology_from_lims(stimulus_ontology_file)
+    else:
+        stimulus_ontology_file = StimulusOntology.DEFAULT_STIMULUS_ONTOLOGY_FILE
+
+    ont = StimulusOntology(ju.read(stimulus_ontology_file))
```

Comment:
It looks like this block of code lines 50-55 are repeated quite a few times in other modules (`run_qc.py`, `run_sweep_extraction.py`) I would try centralizing this into a function unless you think it might change differently for those other modules.